data/autoyast_opensuse/autoyast_btrfs_quota.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+    <bootloader>
+        <global>
+            <timeout config:type="integer">-1</timeout>
+        </global>
+    </bootloader>
+    <general>
+        <mode>
+            <confirm config:type="boolean">false</confirm>
+        </mode>
+    </general>
+    <login_settings>
+        <autologin_user>bernhard</autologin_user>
+    </login_settings>
+    <networking>
+        <keep_install_network config:type="boolean">true</keep_install_network>
+    </networking>
+    <software>
+        <install_recommended config:type="boolean">true</install_recommended>
+        <products config:type="list">
+            <product>openSUSE</product>
+        </products>
+        <patterns config:type="list">
+            <pattern>apparmor</pattern>
+            <pattern>base</pattern>
+        </patterns>
+    </software>
+    <users config:type="list">
+        <user>
+            <fullname>Bernhard M. Wiedemann</fullname>
+            <encrypted config:type="boolean">true</encrypted>
+            <user_password>nots3cr3t</user_password>
+            <username>bernhard</username>
+        </user>
+        <user>
+            <encrypted config:type="boolean">false</encrypted>
+            <user_password>nots3cr3t</user_password>
+            <username>root</username>
+        </user>
+    </users>
+    <partitioning t="list">
+        <drive t="map">
+            <device>/dev/vda</device>
+            <disklabel>gpt</disklabel>
+            <enable_snapshots t="boolean">true</enable_snapshots>
+            <partitions t="list">
+                <partition t="map">
+                    <create t="boolean">true</create>
+                    <format t="boolean">false</format>
+                    <partition_id t="integer">263</partition_id>
+                    <partition_nr t="integer">1</partition_nr>
+                    <resize t="boolean">false</resize>
+                    <size>8388608</size>
```

Comment:
Don't you mind to use human readable values for the partitions?

Reply:
Done. Waiting for VRs to be finished.